Pod/Classes/Acknowledgements/AcknowledgementsViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  AcknowledgementsViewController.swift
+//  Swiftilities
+//
+//  Created by Michael Skiba on 11/21/16.
+//  Copyright © 2016 Raizlabs Inc. All rights reserved.
+//
+
+import UIKit
+
+open class AcknowledgementsViewController: UITableViewController {
+
+    fileprivate static let reuseID = "entryCell"
+
+    public enum LocalizedStrings {
+        public static let acknowlegementsTitle = NSLocalizedString("Acknowlegments", comment: "Default title for the Acknowlegements view controller from Swiftilities")
+    }
+
+    open var viewModel: AcknowledgementsViewModel = AcknowledgementsViewModel(title: "", acknowledgements: []) {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+
+    open var licenseFormatter: (String) -> NSAttributedString = AcknowledgementViewController.defaultLicenseFormatter
+
+    open var licenseViewBackgroundColor: UIColor?
+
+    open var cellBackgroundColor: UIColor? {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+
+    open var registeredCellClass: UITableViewCell.Type = UITableViewCell.self {
+        didSet {
+            updateCellRegistration()
+        }
+    }
+
+    public convenience init(title: String = LocalizedStrings.acknowlegementsTitle,
+                             viewModel: AcknowledgementsViewModel,
+                             licenseViewBackgroundColor: UIColor? = nil,
+                             cellBackgroundColor: UIColor? = nil,
```

Comment:
Any chance to use `UIAppearance` for these instead of explicit configurations? It simplifies the customization story imo.